### ISSUE TITLE
LoadScript would benefit from being a Fragment

### ISSUE THREAD

**Issue by Lepozepo** [JustFly1984/react-google-maps-api#61]
https://github.com/JustFly1984/react-google-maps-api/blob/dfadcfc19b451de819185dbb5e6dd62cc976ff3a/packages/react-google-maps-api/src/LoadScript.tsx#L200

**Comment by FredyC** [comment]
You cannot attach ref to fragment ;)

**Comment by Lepozepo** [comment]
Ah, I see, well regardless that `div` is a pain to deal with. You could do something like this instead maybe without having to refactor too much:

```
render() {
  ...
  return (
    <>
      <div ref={this.check} />
      {this.state.loaded ? this.props.children : this.props.loadingElement}
    </>
  );
}
```

**Comment by Lepozepo** [comment]
Considering it's only being used for [clean up](https://github.com/JustFly1984/react-google-maps-api/blob/dfadcfc19b451de819185dbb5e6dd62cc976ff3a/packages/react-google-maps-api/src/LoadScript.tsx#L88) though, I think you can get rid of the ref and just check for the globally scoped variable that it adds instead? `if (window.google) delete window.google`, I'm still getting familiar with this repo, I just used it this morning ^_^

**Comment by JustFly1984** [comment]
We can't really make it happen. I think we can add optional className property to LoadScript, until there is better solution. Does somebody have something against that?

**Comment by uriklar** [comment]
> Ah, I see, well regardless that `div` is a pain to deal with. You could do something like this instead maybe without having to refactor too much:
> 
> ```
> render() {
>   ...
>   return (
>     <>
>       <div ref={this.check} />
>       {this.state.loaded ? this.props.children : this.props.loadingElement}
>     </>
>   );
> }
> ```

@JustFly1984 are you sure this wouldn't work? I think we need to try it out...

**Comment by JustFly1984** [comment]
I was blind an now I see) I've pushed commit alpha5 branch, it will be there to test tomorrow

**Comment by FredyC** [comment]
I am confused right now. What is the actual problem with that div and what does that fragment solve? I haven't honestly tried this lib yet, so I am just curious.

**Comment by uriklar** [comment]
The problem is that in order to load the google maps script we currently use a wrapper div. The reason for the div is so we can assign a ref on it (to be used later in the cleanup function). I'm not sure the cleanup solution here is the best solution and if we even need this ref, but right now this is the way it works. 

The problem is that this unnecessary div might interfere with a user's html structure. Adding the Fragment allows to render the div not as a wrapper of the map components, but as a sibling. We could also use a protal.. Many ways to solve this issue, but this one looks simple enough

**Comment by JustFly1984** [comment]
Ok, currently last alpha has this fragment fix, so I'm closing an issue. Fix will be included to 1.2.0
